tests/robustness/traffic/etcd.go

```diff
@@ -61,6 +61,18 @@ var (
 			{choice: LargePut, weight: 5},
 		},
 	}
+	// Issue17780EtcdPutDelete is to create high chance to have more delete
+	// requests so that it's likely to compact that revision which is tombstone.
+	Issue17780EtcdPutDelete = etcdTraffic{
```

Comment:
Could this be inline if it is not used by any other tests?

Reply:
You mean inline is about anonymous value during assignment?
`etcdTraffic` is unexported type and `choiceWeight` is alsog unexported type...
I think it's hard to make it inline if I understand it correctly.

Reply:
I see. How about just name it to `EtcdPutDeleteHeavy`?

Reply:
+1 incorporating it with exploratory testing.

Reply:
Hi @siyuanfoundation  @serathius , I want to keep it with special name here because it's hard to reproduce it without special input or setting. If we put it into exploratory testing, existing case might have chance to reproduce it. But it's unknown for us. I was trying many combinations of traffic but it runs one day without luck. And then we introduce compact traffic, it also makes it hard to trigger issue 17780. 

If you have better idea, please let me know. Thanks

Reply:
I would prefer to avoid having a issue specific traffic, to prevent regressions in robustness we don't need 100% reproducability, about 2-5% is ok.

Reply:
I was trying to use existing traffic to reproduce it. Unfortunately, I run it for 3 days. No luck :(

Reply:
let me take a look at it